lib/fastPath.js

```diff
@@ -15,18 +15,17 @@ function pick(arr) {
     }
     let obj = {};
     arr.forEach(entry => obj[entry] = true);
-    return ((entry, idx) => idx in obj) ;
+    return ((entry, idx) => obj.hasOwnProperty(idx));
 }
```

Comment:
@pvenkatakrishnan but `in` will return `true` if `idx` happens to be an inherited property.